test/plot-schema.json

```diff
@@ -16238,7 +16238,7 @@
      "valType": "number"
     },
     "fillcolor": {
-     "description": "Sets the fill color. Defaults to a half-transparent variant of the line color, marker color, or marker line color, whichever is available.",
+     "description": "Sets the fill color. Defaults to a half-transparent variant of the line color, marker color, or marker line color, whichever is available. If fillgradient is specified, fillcolor is ignored except for setting the background color of the hover label, if any.",
```

Comment:
This one is the `box` trace description. Do we want to support this for this trace?

Reply:
> I was also surprised by the many places where the fillcolor description was changed
in the newly generated schema: not only for scatter traces but apparently in many
other places as well, even though I only edited the attributes.js in traces/scatter.
I am not sure these are all correct, please advise.

I don't know why these all got updated or what to do to prevent it..

Reply:
Alternatively you could make a function in scatter and export it with options to create different descriptions.

Reply:
I'd be happy to help fix this part if you want?

Reply:
yes, please go ahead

Reply:
Please pull my commits from [`scatter-gradient-fills_dev`](https://github.com/plotly/plotly.js/compare/scatter-gradient-fills_dev) branch.

Reply:
I'd love to include this feature in the minor release today. Would you be able to pick my commits from https://github.com/plotly/plotly.js/compare/scatter-gradient-fills_dev ?
Please let me know. Thank you!


> Please pull my commits from [`scatter-gradient-fills_dev`](https://github.com/plotly/plotly.js/compare/scatter-gradient-fills_dev) branch.

Reply:
sorry, I didn't have time to look at this recently. I have merged your changes now, but I think we still have some points to resolve below anyways before this can go into a release, right?